lib/validateMessage.js

```diff
@@ -21,6 +21,45 @@ var error = function() {
 
 exports.config = config;
 
+var validateScope = function(isValid, scope) {
```

Comment:
nit: We're not doing linting (yet), but I prefer FunctionDeclarations over FunctionExpressions (so this would be: `function validateScope(isValid, scope) { ... }`)